test/data/noConflict.html

```diff
@@ -0,0 +1,31 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+	<meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
+	<script>var QUnit = parent.QUnit</script>
+	<script src="testinit.js"></script>
+	<script src="../../src/sizzle.js"></script>
+	<script>
+		var Sizzle1 = Sizzle;
+	</script>
+	<script src="../../src/sizzle.js"></script>
+	<script>
+		var Sizzle2 = Sizzle;
+	</script>
+</head>
+<body>
+	<script>
+		var doc = parent.document,
+			unframed = [ doc.getElementById( "qunit-fixture" ), doc.body, doc.documentElement ],
+			framed = Sizzle( "*" );		
+
+		window.parent.iframeCallback( function( QUnit ) {
```

Comment:
Passing a function to `iframeCallback` is really cool. As for the body, see https://github.com/jquery/sizzle/issues/352#issuecomment-151511997 . I'd expect something like:
```js
QUnit.deepEqual( selection, [ document.documentElement.firstChild, document.body ], "selection sanity check" );
QUnit.notStrictEqual( Sizzle, Sizzle1, "first Sizzle overwritten by second" );
QUnit.deepEqual( Sizzle( "html > *" ), Sizzle1( "html > *" ), "second Sizzle selects identically" );
var replaced = Sizzle.noConflict();
QUnit.strictEqual( Sizzle, Sizzle1, "first Sizzle restored by noConflict" );
QUnit.strictEqual( replaced, Sizzle2, "second Sizzle returned by noConflict" );
QUnit.deepEqual( Sizzle( "html > *" ), Sizzle2( "html > *" ), "both Sizzles still select identically" );
QUnit.deepEqual( Sizzle( "html > *" ), selection, "Sizzle still selects correctly" );
```